game_eggs/steamcmd_servers/palworld/README.md

```diff
@@ -98,4 +98,21 @@ As of January 19th 2024 the server requires about 5GB of storage capacity. This
 | Game            | 8211    |
 | RCON (optional) | 25575   |
 
-The RCON port does not need to be allocated.
\ No newline at end of file
+The RCON port does not need to be allocated.
+
+### Updating
```

Comment:
We could make this a standalone doc in the repo named `UPDATING_EGGS.md` or something.

The parser docs are fine to keep.

Reply:
I second this.

Reply:
I'm not sure if it makes sense to new users to simply say "Update your egg". Does this mean creating a new egg in the nest?